homework/Contributor.js

```diff
@@ -13,7 +13,21 @@ class Contributor {
    * @param {HTMLElement} container The container element in which to render the contributor.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const logi = this.contributor.login;
+    const img = this.contributor.avatar_url;
+    const link = this.contributor.html_url;
+    const contribute = this.contributor.contributions;
+    const li = Util.createAndAppend('li', container, { class: 'right-li' });
+
+    const contributorsList = `<a target=_blank href=${link}><img src=${img}><br>${logi}<br>Contributions: ${contribute}</a>`;
+
+    li.innerHTML += contributorsList;
+
+    li.addEventListener('keyup', contributor => {
```

Comment:
Hi Darwesh - may I ask why did you add this event listener here?

Reply:
Hello @yash-kapila ,
Thank you for your feedback and your useful advises.
I add this event listener here to do this part from the homework:

 (a user should be able to navigate through all interactive elements using the keyboard (e.g., using the Tab key). Pressing Enter on such an element should be equivalent to clicking the mouse.)

I just check how to do it on google and i copy and past the code and it is work just fine .
Is there any wrong with it ?
Best regards,

Reply:
Ah no. There isn't anything wrong with it but keyboard navigation would work in this case even without it. Can you check? 🙂 

If you have anchor tags on the page, then browser automatically moves to them as you navigate through the page using keyboard.